analyzer/node_stats/node_stats.go

```diff
@@ -44,9 +44,9 @@ func NewAnalyzer(
 		cfg.Interval = 3 * time.Second
 	}
 	logger = logger.With("analyzer", nodeStatsAnalyzerName)
-	// Default to [consensus, emerald, sapphire, pontusx] if layers is not specified.
+	// Default to [consensus, emerald, sapphire, pontusx_dev] if layers is not specified.
```

Comment:
Why dont we also default to pontusx_test?

Reply:
I hesitated because `pontusx_test` is only on testnet, but then decided to add it anyway. At worst, this default config will lead to a bit of additional error spam.